src/components/formatters/FormattedNumber.js

```diff
@@ -1,17 +1,28 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const FormattedNumber = props => (
+const FormattedNumber = ({
+    value,
+    minimumFractionDigits,
+    maximumFractionDigits,
+}) => (
     <span>
         {new Intl.NumberFormat(sessionStorage.getItem('uiLocale'), {
-            minimumFractionDigits: 2,
-            maximumFractionDigits: 2,
-        }).format(props.value)}
+            minimumFractionDigits,
+            maximumFractionDigits,
+        }).format(value)}
     </span>
 )
 
+FormattedNumber.defaultProps = {
+    minimumFractionDigits: 0,
```

Comment:
Hendrik and I talked about this as well. Seems like the default has changed here. Just commenting so other reviewers know that this was discussed already.